lib/engines/content_block_manager/app/models/content_block_manager/content_block/version/changed_field.rb

```diff
@@ -0,0 +1,6 @@
+module ContentBlockManager
```

Comment:
I don't like the name `ChangedField` - would `EditionFieldDiff` be better? or something else??

Reply:
I think maybe if you embed it in the `ContentBlock::Edition` namespace, so it's `ContentBlockManager::ContentBlock::Edition::FieldDiff` maybe?